graphql-dxm-provider/src/main/java/org/jahia/modules/graphql/provider/dxm/node/GqlJcrMutationSupport.java

```diff
@@ -116,6 +116,29 @@ public static List<JCRPropertyWrapper> setProperties(JCRNodeWrapper node, Collec
         }
     }
 
+    /**
+     * Delete the provided properties to the specified node.
+     *
+     * @param node       The JCR node to delete properties from
+     * @param properties the collection of properties to be deleted
+     * @return Boolean
+     */
+    public static boolean deleteProperties(JCRNodeWrapper node, Collection<GqlJcrPropertyI18nInput> properties) {
+        if (properties == null) {
+            return false;
+        }
+
+        for (GqlJcrPropertyI18nInput prop : properties) {
+            try {
+                NodeHelper.getNodeInLanguage(node, prop.getLanguage()).getProperty(prop.getName()).remove();
```

Comment:
The way we're handling exceptions here might leave operation in a half-way state e.g. if one of them missing a language value, we end up with some props deleted and some aren't (even if rest could still be valid)

Maybe better option here is to handle exceptions individually and collect them as a single exception. This way we process all valid properties and throw an exception with list of the names not processed.

Reply:
ok, perfectly fine in that case.